### PR TITLE
M23 rebase

### PR DESCRIPTION
**Pull request by washort**
Upstream changes were fairly minor this round, only thing of note was the #3291 schema viewer stuff conflicting with ed2ac40

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -107,47 +83,87 @@ jobs:
           path: /tmp/artifacts/
   build-docker-image:
     docker:
-      - image: circleci/node:8
+      - image: circleci/buildpack-deps:xenial
```
Comment (jezdez) [inline .circleci/config.yml]: This needs npm below when running `npm run bundle`, we've installed npm inside a `buildpack-deps:xenial` container [in the last rebase](https://github.com/mozilla/redash/blob/b42623e2a106a9e3ec1dbc94715b433ee0b3ad51/.circleci/config.yml#L84-L119) but we can also keep using node:8 and just `sudo apt install python-pip` to get pip which would be more in line with what upstream has.

File: .circleci/config.yml [hunk 2]
```diff
@@ -107,47 +83,87 @@ jobs:
...
       - setup_remote_docker
       - checkout
-      - run: sudo apt install python-pip
       - run: sudo pip install -r requirements_bundles.txt
-      - run: .circleci/update_version
+      - run: .circleci/update_version "master"
       - run: npm run bundle
-      - run: .circleci/docker_build
+      - run: .circleci/docker_build "master"
+  build-docker-image-rc:
```
Comment (jezdez) [inline .circleci/config.yml]: Those `build-docker-image*` jobs would need to follow the same pattern as the `build-docker-image` job above.

File: bin/bundle-extensions [hunk 3]
```diff
@@ -78,7 +78,7 @@ def load_bundles():
 
     """
     bundles = odict()
-    for entry_point in entry_points().get("redash.bundles", []):
+    for entry_point in entry_points().get("redash.extensions", []):
```
Comment (jezdez) [inline bin/bundle-extensions]: This is a wrong change again.

File: package.json [hunk 4]
```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "node --max-http-header-size=16385 ./node_modules/webpack-dev-server/bin/webpack-dev-server.js",
-    "bundle": "bin/bundle-extensions",
+    "bundle": "PYTHONPATH=. bin/bundle-extensions",
```
Comment (jezdez) [inline package.json]: This isn't needed.

**Comment by washort** [comment]
I didn't see that `release` had not been updated. I've redone this from `release-m22.1` and I believe it addresses your concerns.

**Comment by washort** [comment]
this is on master now.

**Pull request closed without merging**